examples/hugectr/criteo-hugectr.ipynb

```diff
@@ -203,32 +325,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import numpy as np\n",
-    "\n",
     "dict_dtypes={}\n",
     "\n",
     "for col in CONTINUOUS_COLUMNS:\n",
     "    dict_dtypes[col] = np.float32\n",
-    "    \n",
-    "for col in CATEGORICAL_COLUMNS:\n",
-    "    dict_dtypes[col] = np.int64\n",
+    " \n",
+    "# Categorical output datatypes are specified in categorify operation\n",
+    "# for col in CATEGORICAL_COLUMNS:\n",
+    "#    dict_dtypes[col] = np.int64\n",
```

Comment:
If we don't need the dtypes here, I'd rather just delete than comment out

Reply:
Makes sense